ocamlformat.opam

```diff
@@ -29,7 +29,7 @@ depends: [
   "dune" {build & >= "1.1.1"}
   "fpath"
   "ocaml-migrate-parsetree" {>= "1.3.1"}
-  "octavius" {>= "1.2.0"}
+  "odoc" {>= "1.4.0"}
```

Comment:
```suggestion
  "odoc" {>= "1.4.1"}
```

just merged https://github.com/ocaml/opam-repository/pull/14349

Reply:
Finally available to `opam update`. Upgraded.